src/components/Action/Action.vue

```diff
@@ -1,3 +1,89 @@
+<script setup lang="ts">
```

Comment:
You might need to disable this rule lint rule

Reply:
Yeah I guess